docs/guides/general-terminology/open-source.mdx

```diff
@@ -0,0 +1,20 @@
+
+---
+
+                                                               title: Open Source
```

Comment:
![Screenshot from 2023-02-01 02-54-17](https://user-images.githubusercontent.com/86586277/215886773-47928d34-3ece-46b8-854b-79eaf67b2b66.png)

This long `title` is getting displayed
use this instead
```md
---
title: Open Source
---
```

Reply:
How can i add credits to the contribution?
Before merging to main let me know this.